proto/cacheclient.proto

```diff
@@ -1025,3 +1028,25 @@ message _SortedSetLengthByScoreResponse {
     _Missing missing = 2;
   }
 }
+
+message _SortedSetUnionStoreRequest {
+  enum AggregateFunction {
+    SUM = 0;
+    MIN = 1;
+    MAX = 2;
+  }
+
+  message _Source {
+    bytes set_name = 1;
+    float weight = 2;
+  }
+
+  bytes set_name = 1;
```

Comment:
nit: can we call it `destination_set_name` for clarity?

Reply:
I think it is implicit that this is the destination by having `Store` in the name of the api and it follows the pattern of other apis